cookbooks/cdo-varnish/libraries/http_cache.rb

```diff
@@ -0,0 +1,95 @@
+# HTTP Cache configuration.
+#
+# See cdo-varnish/README.md for more information on the configuration format.
+class HttpCache
+  STATIC_ASSETS = {
+    # For static-asset extensions, don't forward any cookies or headers.
+    path: %w(cur pdf png gif jpeg jpg ico mp3 swf css js).map{|ext| "*.#{ext}"},
+    headers: [],
+    cookies: 'none'
+  }
+
+# Host header is needed for Varnish to delegate to pegasus/dashboard depending on the host.
+# Accept-Language header is needed to cache language-specific pages.
+  ALL_HEADERS = %w(Host Accept-Language)
+  LANGUAGE_COOKIE = %w(language_)
+
+# HTTP-cache configuration that can be applied both to CDN (e.g. Cloudfront) and origin-local HTTP cache (e.g. Varnish).
+# Whenever possible, the application should deliver correct HTTP response headers to direct cache behaviors.
+# This hash provides extra application-specific configuration for whitelisting specific request headers and
+# cookies based on the request path.
+  def self.config(session_key, storage_id)
+    all_cookies = [
```

Comment:
How about calling this hoc_cookies or whitelisted_cookies?

all_cookies is confusing when I read it below because it doesn't mean *all* cookies, it mean's certain whilisted cookies.